package.json

```diff
@@ -34,6 +34,7 @@
     "build/contracts/SnappBase.json",
     "build/contracts/SnappBaseCore.json",
     "build/contracts/TokenConservation.json",
+    "build/contracts/ERC20.json",
```

Comment:
I think its better to include the `IERC20` interface here instead, this will also keep the package smaller as it has no bytecode and has the same ABI.

Reply:
good point.
good for me!